README.md

```diff
@@ -1,70 +1,88 @@
-# Getting Started with Create React App
+# Project Information
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
-
-## Available Scripts
 
-In the project directory, you can run:
+    This project is frontend part of the STONKS application.
+    Stonks is app that will have capability to predict stock movement based of LR model.
+    Users are able to see current stock parameters.
+    In future crypto may also be supported.
 
-### `yarn start`
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
-
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+## List of Moderators
 
-### `yarn test`
+* **[Mayank Talwar](https://github.com/DreadArceus)**
```

Comment:
Add everything to him that others contain except GitHub. That is exclusively for me :)

Reply:
Done

Reply:
Not done. Push changes if done locally